docs/software/admin.md

```diff
@@ -969,6 +970,83 @@ At this point `info` will tell you that the node is setup to vote for this upgra
       ]
 ```
 
+## Overlay topology survey
```

Comment:
I think this should be under `Overlay information` and not a top level section.

Also, you should link to that section from the end of `Configuration basics` with a caption `Auditing of the P2P network is enabled by default, see the "overlay topology" section for more detail if you'd like to disable it`